examples/basic_cogs/run.py

```diff
@@ -1,17 +1,17 @@
-from glob import glob
-
 from pincer import Client
 
+from cogs import OnReadyCog, SayCog, ErrorHandler
+
 
 class Bot(Client):
     def __init__(self, *args, **kwargs):
         self.load_cogs()
         super().__init__(*args, **kwargs)
 
     def load_cogs(self):
-        """Load all cogs from the `cogs` directory."""
-        for cog in glob("cogs/*.py"):
-            self.load_cog(cog.replace("/", ".").replace("\\", ".")[:-3])
+        self.load_cog(OnReadyCog)
+        self.load_cog(SayCog)
+        self.load_cog(ErrorHandler)
```

Comment:
Implement a `load_cogs` that would allow the following?
```suggestion
        super().load_cogs(OnReadyCog, SayCog, ErrorHandler)
```

might just very be 
```py
def load_cogs(self, *cogs):
    for cog in cogs:
        self.load_cog(cog)
```